test/unit/spec/MfaVerify_spec.js

```diff
@@ -174,6 +174,39 @@ function (Okta,
       });
     }
 
+    function setupWebauthnOnly() {
```

Comment:
The other setup function has `Expect.waitForMfaVerify()`, and for this particular test, where Windows Hello is the only factor, it never goes to that controller.